packages/nextjs/app/builders/0xeb50dD3Bb9E4F8986eB59A3fFbC9D72a4A3DD1c8/page.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from "next/image";
+import Link from "next/link";
+import avatar from "./avatar.png";
+import styles from "./styles.module.css";
+import type { NextPage } from "next";
+
+const Personal: NextPage = () => {
```

Comment:
Since there are more personal pages, it would be nice to give it a more descriptive name.